cmd/template/cluster/flag.go

```diff
@@ -78,6 +84,33 @@ func (f *flag) Init(cmd *cobra.Command) {
 func (f *flag) Validate() error {
 	var err error
 
+	if f.ClusterID != "" {
+		if len(f.ClusterID) != key.IDLength {
+			return microerror.Maskf(invalidFlagError, "--%s must be length of %d", flagClusterID, key.IDLength)
+		}
+
+		if _, err := strconv.Atoi(f.ClusterID); err == nil {
+			// string is numbers only, which we want to avoid
+			return microerror.Maskf(invalidFlagError, "--%s must be alphanumeric", flagClusterID)
+		}
+
+		matched, err := regexp.MatchString("^[a-z]+$", f.ClusterID)
+		if err == nil && matched == true {
+			// strings is letters only, which we also avoid
+			return microerror.Maskf(invalidFlagError, "--%s must be alphanumeric", flagClusterID)
+		}
+
+		if strings.ContainsAny(f.ClusterID, "1l") {
```

Comment:
Looks like this is covered by the case below. Is this just to have a nicer error message for these two characters in particular?

Reply:
yes as I was going step by step, but you right it's overhead